unfetter-discover/unfetter-ui
# Issue 1121/delete baseline

Fixes unfetter-discover/unfetter#1121

Deletion of baselines now allowed from three-dot menu in sidebar as well as baseline master list.

## To test:
1. Pull this PR
2. Restart UI:  `docker-compose restart unfetter-ui`
3. Delete any baseline by clicking the delete icon in the baseline master list or selecting `Delete` in the three-dot menu for an open baseline
4. Confirmation dialog should appear prior to deletion, citing the name of the baseline
